apps/src/lib/levelbuilder/rubrics/LearningGoalItem.jsx

```diff
@@ -30,14 +30,22 @@ export default function LearningGoalItem({deleteItem}) {
                 />
               </label>
             </div>
-            <label>
+            <label style={styles.labelAndInput}>
               Use AI to assess
               <input
```

Comment:
This may be out of scope as we already use a checkbox here but I'm curious if using the checkbox from the [component library](https://github.com/code-dot-org/code-dot-org/tree/staging/apps/src/componentLibrary/checkbox) works here?

Reply:
That might work. I wasn't aware that we already had a checkbox component. I'll try it.

Reply:
It looks like the existing checkbox component doesn't allow color changing or other styling.